content/manuals/build/bake/remote-definition.md

```diff
@@ -180,6 +164,26 @@ docker buildx bake -f bake.hcl -f cwd://local.hcl "https://github.com/crazy-max/
 }
 ```
 
+One case where combining local and remote Bake definitions becomes necessary is
+when you're building with a remote Bake definition in GitHub Actions and want
+to use the [metadata-action](https://github.com/docker/metadata-action) to
+generate tags, annotations, or labels. The metadata action generates a Bake
+file available in the runner's local Bake execution context. To use both the
+remote definition and the local "metadata-only" Bake file, specify both files
+and use the `cwd://` prefix for the metadata Bake file:
+
+```yml
+      -
+        name: Build
+        uses: docker/bake-action@v4
+        with:
+          source: "${{ github.server_url }}/${{ github.repository }}.git#${{ github.ref }}"
```

Comment:
marking this one for https://github.com/docker/bake-action/pull/193 to remember we need to remove that as follow-up

Reply:
Fyi, `source: "{{defaultContext}}"` also works: https://github.com/docker/bake-action/blob/cdf7e1c5cbb6b6a740f80bfe3ed7939f5c30424f/.github/workflows/ci.yml#L419

https://github.com/docker/bake-action/blob/cdf7e1c5cbb6b6a740f80bfe3ed7939f5c30424f/src/context.ts#L136-L138